### PR TITLE
fix for values < 1

### PR DESCRIPTION
**Pull request by obama**
now works with values < 1 too

### CODE REVIEW

File: src/leaflet.nauticscale.js [hunk 1]
```diff
@@ -22,14 +22,33 @@ L.Control.ScaleNautic = L.Control.Scale.extend({
...
-			nauticalMiles = L.Control.Scale.prototype._getRoundNum.call(this, maxNauticalMiles);
-		} else {
-			nauticalMiles = maxNauticalMiles > 0.1 ? Math.round(maxNauticalMiles * 10) / 10 : Math.round(maxNauticalMiles * 100) / 100;
-		}
+		nauticalMiles = this._getRoundNum(maxNauticalMiles);
 
-		scale.style.width = Math.round(this.options.maxWidth * (nauticalMiles / maxNauticalMiles)) - 10 + 'px';
-		scale.innerHTML = nauticalMiles + ' nm';
+		scale.style.width = Math.round(this.options.maxWidth * (nauticalMiles / maxNauticalMiles)) + 'px';
+		scale.innerHTML = nauticalMiles + ' nmi';
```
Comment (wattnpapa) [inline src/leaflet.nauticscale.js]: Hmmm... this is now an interesting point.
There are some abbreviation for the Nautical Mile.

Maybe we should make the configurable depending of the needs of the user.

Wikipedia show us the Problem: https://en.wikipedia.org/wiki/Nautical_mile#Unit_symbol

We should use a Default and if someone want to have something else, it should be possible to do this over an option:

map.addControl(new L.Control.ScaleNautic({
        metric: true,
        imperial: true,
        nautic: true,
        nauticUnit: "nmi"
    }));

Comment (obama) [reply src/leaflet.nauticscale.js]: yes, good idea. i didn't even consider this.

Comment (wattnpapa) [reply src/leaflet.nauticscale.js]: hey @obama 
can we split this ?
remove this and the I will accept the pull request

after that we can add the nauticUnit as an Option ;)

**Comment by obama** [comment]
the Travis CI fails because some phantomjs thing or so?

**Comment by wattnpapa** [comment]
yes there was a problem with travis 
I just updated the travis file no it should work

**Pull request merged**